src/gui/tray/usermodel.h

```diff
@@ -109,14 +131,18 @@ public slots:
     void slotRebuildNavigationAppList();
     void slotSendReplyMessage(const int activityIndex, const QString &conversationToken, const QString &message, const QString &replyTo);
     void forceSyncNow() const;
+    void slotCapabilitiesChanged();
+    void slotFetchGroupFolders();
 
 private slots:
     void slotPushNotificationsReady();
     void slotDisconnectPushNotifications();
     void slotReceivedPushNotification(OCC::Account *account);
     void slotReceivedPushActivity(OCC::Account *account);
     void slotCheckExpiredActivities();
-
+    void slotGroupFoldersFetched(QNetworkReply *reply);
+    void parseNewGroupFolderPath(const QString &path);
+    void prePendGroupFoldersWithLocalFolder();
```

Comment:
same question, why a slot ?

Reply:
> 

@mgallien because you and @claucambra Insisted in the past that we mast make all void methods slots